src/AppInstallerCLICore/ExecutionReporter.cpp

```diff
@@ -209,6 +211,12 @@ namespace AppInstaller::CLI::Execution
         }
     }
 
+    void Reporter::RestoreDefault() 
+    {
+        EndProgress(true);
+        m_standardOutputStream.Close();

```

Comment:
I don't think that a function called `RestoreDefault` should close the output stream.  Without knowing, I would think that it resets the state, not that it prevents future output.

Reply:
The function name is the one thing I see changing here to something that clearly indicates that we're closing the output stream, since every write to output stream happens through the reporter. Also, restoring the default doesn't prevent a background asynchronous operation from altering the output stream. The goal is that if we ever get to this point, the reporter is about to be terminated and we should close.